active_model_otp.gemspec

```
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 2.3"
 
   spec.add_dependency "activemodel"
-  spec.add_dependency "rotp", "~> 5.0.0"
+  spec.add_dependency "rotp", "~> 5"
```

In case of needing to upgrade for security reason on 5.0, why we just bump to the next one? do you know what are the versions impacted from 5.0?
 
```suggestion
  spec.add_dependency "rotp", "~> 5.1.0"
```